python/cugraph/tests/test_bfs.py

```diff
@@ -250,7 +242,7 @@ def test_bfs(graph_file, directed, seed):
                 seed=seed)
 
 
-@pytest.mark.parametrize('graph_file', DATASETS)
+@pytest.mark.parametrize('graph_file', utils.DATASETS)
```

Comment:
Should this be DATASETS_5

Reply:
To match what was there yes, but to move every test to using a common set then no.